tcl/register.tcl

```diff
@@ -68,4 +74,40 @@ proc qc::validate {method path proc_args proc_body} {
     qc::nsv_dict set handlers VALIDATE $method $path args $args 
     qc::nsv_dict set handlers VALIDATE $method $path body $proc_body 
     qc::nsv_dict set handlers VALIDATE $method $path defaults $defaults
+}
+
+proc qc::registered {method url_path} {
+    #| Checks if the given method url_path is registered for the given filter.
+    set method [string toupper $method]
+    if { [qc::nsv_dict exists registered $method] } {
+        return [qc::pattern_matches $url_path [qc::nsv_dict get registered $method]]
+    } else {
+        return false
+    }
+}
+
+proc qc::pattern_matches {url_path patterns} {
+    #| Checks if the given url_path matches any of the given patterns.
```

Comment:
There's likely a better name for this proc:

* `qc::path_matches_pattern`
* `qc::url_pattern_matches`
* `qc::url_patterns_match`
* `qc::matches_patterns`
* `qc::path_matches_patterns`
* `qc::path_patterns_check`
* `qc::check_path_patterns`